docs/modules/ROOT/pages/index.adoc

```diff
@@ -17,21 +17,21 @@ for tables having Change Data Capture (CDC) enabled to https://www.datastax.com/
 {cdc_cass} has two components:
 
 * DataStax Change Agent for Apache Cassandra, which is an event producer deployed as a JVM agent on each Cassandra data node
-* {csc_pulsar_first}, which is a source connector deployed in your streaming platform
+* {cdc_pulsar}, which is a source connector deployed in your streaming platform
 
 The following diagram describes the general architecture.
 
 image::cdc-for-cassandra-overview.png[]
 
 Since version 3.0, Cassandra has included a change data capture (CDC) feature.
-The CDC feature can be enabled on the table level by setting the table property cdc=true, after which any commit log containing data for a CDC-enabled table will be moved to the CDC directory specified in `cassandra.yaml` on discard/flush (default: `cdc_raw`).
+The CDC feature can be enabled on the table level by setting the table property `cdc=true`, after which any commit log containing data for a CDC-enabled table will be moved to the CDC directory specified in `cassandra.yaml` on discard/flush (default: `cdc_raw`).
 Since Cassandra 4.0, commit logs can be synced to the `cdc_raw` directory periodically (by default, 10 seconds).
 
 When CDC is enabled:
 
 . The change agent running on each node reads the Cassandra `commitlog` files in the `cdc_raw` directory and emits a message for each Cassandra mutation on a CDC-enabled table to a per-table events topic.
-. The {csc_pulsar} processes these events and fetches the mutated Cassandra row.
-. The {csc_pulsar} writes the row into the data topic. This ensures the most recent state of the Cassandra table is replicated into the data topic without the need to time re-order Cassandra mutations.
+. The {cdc_pulsar} processes these events and fetches the mutated Cassandra row.
+. The {cdc_pulsar} writes the row into the data topic. This ensures the most recent state of the Cassandra table is replicated into the data topic without the need to time re-order Cassandra mutations.
```

Comment:
The CSC connector writes the row into the data topic with necessary processing tasks that make sure the most recent state of the Cassandra table is replicated into the data topic correctly (no duplication, right order, etc.)